### ISSUE TITLE
Kefrens 174 Bobs

### ISSUE THREAD

**Issue by chris70c** [dirkwhoffmann/vAmiga#547]
Bitplanes seem to be all over the place in this one.
![vamiga](https://user-images.githubusercontent.com/3858768/134279376-a4f33140-fb0d-484c-a002-89cbae93c66e.png)
![winuae](https://user-images.githubusercontent.com/3858768/134279379-9d5521d3-104a-4223-8117-748ead6e4d34.png)
[Kefrens_TA_TST-174BobsOF.zip](https://github.com/dirkwhoffmann/vAmiga/files/7207532/Kefrens_TA_TST-174BobsOF.zip)

**Comment by dirkwhoffmann** [comment]
It might be an OCS / ECS issue. It works well with ECS Agnus:

<img width="853" alt="Bildschirmfoto 2021-09-22 um 11 41 00" src="https://user-images.githubusercontent.com/12561945/134329848-e88b7821-a244-4121-969f-13684677d329.png">

Did you run it in UAE with OCS Agnus?

**Comment by chris70c** [comment]
Yes all tests in WinUAE are done with the quickstart configuration OCS 512+512 KS1.3

**Comment by dirkwhoffmann** [comment]
Got it. It's a duplicate of #543. It runs just fine with the PLCC OCS Agnus:

<img width="853" alt="Bildschirmfoto 2021-09-22 um 17 36 32" src="https://user-images.githubusercontent.com/12561945/134385443-da3fc34a-f464-4bcc-a63b-00da1e9ebe96.png">

If the older DIP Agnus (A1000 Agnus) is selected, the graphics glitches occur.
